war/src/main/js/components/combo-box/index.js

```diff
@@ -0,0 +1,97 @@
+import behaviorShim from "@/util/behavior-shim";
+import Utils from "@/components/dropdowns/utils";
+
+function init() {
+  function convertSuggestionToItem(suggestion, e) {
+    const confirm = () => {
+      e.value = suggestion.name;
+      e.focus();
```

Comment:
It might make sense to trigger form validation here, as done in #9453 (picking item from combobox may have fixed a previous validation error)